src/flownet/config_parser/_config_parser.py

```diff
@@ -241,6 +241,19 @@ def _to_abs_path(path: Optional[str]) -> str:
                                                     },
                                                 },
                                             },
+                                            "WSPR": {
```

Comment:
The other vectors have both rate and cumulative. Does it make sense to add WSPT here as well?